conf/config.go

```diff
@@ -106,6 +106,14 @@ type DNSSeed struct {
 	BPCount        int      `yaml:"BPCount"`
 }
 
+type MQTTBrokerInfo struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type MQTTBrokerInfo should have comment or be unexported